runners/spark/src/test/java/org/apache/beam/runners/spark/ProvidedSparkContextTest.java

```diff
@@ -95,7 +97,9 @@ private void testWithValidProvidedContext(JavaSparkContext jsc) throws Exception
         PAssert.that(output).containsInAnyOrder(EXPECTED_COUNT_SET);
 
         // Run test from pipeline
-        p.run().waitUntilFinish();
+        PipelineResult result = p.run();
+
+        TestPipeline.verifyPAssertsSucceeded(p, result);
```

Comment:
Any reason not to make this an instance method like `verifyPAssertsSucceeded()`?